integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/EmployeePortalTestCase.java

```diff
@@ -94,4 +140,234 @@ public void cleanup() {
         } catch (Exception ignored) {
         }
     }
+
+    /**
+     * Get client id and client secret needed to generate oauth token to access apim.
+     *
+     * @return a concatenated string comprising client id and client secret delimited by a colon
+     * @throws NoSuchAlgorithmException if sendPost method fails
+     * @throws IOException              if sendPost method fails
+     * @throws KeyManagementException   if sendPost method fails
+     */
+    private String getClientIdColonClientSecret() throws NoSuchAlgorithmException
+            , IOException, KeyManagementException {
+        // Get client ID and client secret for alice user
+        String url = "https://wso2-apim/client-registration/v0.14/register";
+        String usernameColonPassword = "alice" + ":" + "alice123";
+        String parameters = "{\"callbackUrl\": \"www.google.lk\", \"clientName\": \"rest_api_store\", \"owner\": " +
+                "\"alice\", \"grantType\": \"password refresh_token\", \"saasApp\": true}";
+        String authenticationKey = Base64.getEncoder().encodeToString(usernameColonPassword.getBytes(
+                StandardCharsets.UTF_8));
+        //Add headers
+        Map<String, String> headers = new HashMap<>();
+        headers.put("Authorization", "Basic " + authenticationKey);
+        headers.put("Content-Type", "application/json");
+        String response = httpClient.sendPost(url, parameters, headers);
+        JsonObject responseJson = new JsonParser().parse(response).getAsJsonObject();
+        Assert.assertTrue(responseJson.isJsonObject());
+        // Get client id and client secret from the response
+        String clientId = responseJson.get("clientId").getAsString();
```

Comment:
ClientId, and ClientSecret should be defined as constants.

Reply:
Updated clientId and clientSecret as constant strings.